.bazelrc

```diff
@@ -192,6 +193,14 @@ build:asan --test_env=KJ_CLEAN_SHUTDOWN=1
 build:asan --copt="-DV8_USE_ADDRESS_SANITIZER"
 build:asan --per_file_copt='external/v8@-DADDRESS_SANITIZER,-DLEAK_SANITIZER'
 
+# address sanitizer with release optimizations – useful for catching bugs that only manifest
+# with higher optimization levels or to test production-like performance with sanitizers.
+# Overrides sanitizer-common's -Og and -g with release-appropriate settings.
+build:asan-release --config=asan
+build:asan-release --copt="-O2"
```

Comment:
Our regular release builds seem to be -O3. Did you intentionally pick -O2 for release-asan?